services/venice-controller/src/main/java/com/linkedin/venice/controller/util/UpdateStoreUtils.java

```diff
@@ -0,0 +1,1215 @@
+package com.linkedin.venice.controller.util;
+
+import static com.linkedin.venice.controllerapi.ControllerApiConstants.ACCESS_CONTROLLED;
+import static com.linkedin.venice.controllerapi.ControllerApiConstants.ACTIVE_ACTIVE_REPLICATION_ENABLED;
+import static com.linkedin.venice.controllerapi.ControllerApiConstants.AMPLIFICATION_FACTOR;
+import static com.linkedin.venice.controllerapi.ControllerApiConstants.AUTO_SCHEMA_REGISTER_FOR_PUSHJOB_ENABLED;
+import static com.linkedin.venice.controllerapi.ControllerApiConstants.BACKUP_STRATEGY;
```

Comment:
The convention (noticed in other places in the codebase as well) is to wildcard static imports.
Would make it less verbose and more readable.

Reply:
We prefer not to use wildcards. The IDEs make it easy to collapse the imports, so, this is not too problematic. The ones that exist need to be removed.